### PR TITLE
oracle: separated devel and precomp headers and moved precomp pcbcfg.…

### PR DESCRIPTION
**Pull request by duperray**
…cfg file to the right folder.
It seems that the path to pcbcfg.cfg file for precomp binary (for instance proc) is hardcoded.
so you need to put this config file here %{oradir}/lib/precomp/admin (note the /lib/)

also all headers where in the devel package with a joker. I separated explicitely the header for the
devel package and for the precomp package (same target dir).

incremented the package version.
Warning only tested on x86_64 arch.

### CODE REVIEW

File: oracle/oracle-instantclient-12.1-x86_64.spec [hunk 1]
```diff
@@ -190,10 +190,9 @@ install -p wrc 		%{buildroot}%{oradir}/bin
 ln -sf %{oradir}/bin/wrc %{buildroot}%{_bindir}/wrc
 
 # Precomp
-# NOTICE: don't install rtsora - broken dep on  libcobcrtn64.so
```
Comment (remicollet) [inline oracle/oracle-instantclient-12.1-x86_64.spec]: Please, keep this comment

File: oracle/oracle-instantclient-12.1-x86_64.spec [hunk 2]
```diff
@@ -284,18 +311,30 @@ rm -rf %{buildroot}
...
 %{_bindir}/proc
 %{_bindir}/procob
+%{incdir}/sqlkpr.h  
+%{incdir}/sqlca.h  
+%{incdir}/sqlcpr.h  
+%{incdir}/sql2oci.h  
+%{incdir}/sqlda.h  
+%{incdir}/sqlucs2.h  
+%{incdir}/oraca.h  
+%{incdir}/sqlapr.h
```
Comment (remicollet) [inline oracle/oracle-instantclient-12.1-x86_64.spec]: Should be in alphabetical order... to make easier the diff with upstream RPM.

File: oracle/oracle-instantclient-12.1-x86_64.spec [hunk 3]
```diff
@@ -242,7 +241,35 @@ rm -rf %{buildroot}
 %{oradir}/lib/libclntsh.so
 %{oradir}/lib/libocci.so
 %{oradir}/lib/ottclasses.zip
-%{incdir}
```
Comment (remicollet) [inline oracle/oracle-instantclient-12.1-x86_64.spec]: Il manque aussi un %dir %{incdir}

**Comment by remicollet** [comment]
Indeed, goot catch up, new layout match upstream one.

File: oracle/oracle-instantclient-12.1-x86_64.spec (hunk 2 above)
Comment (duperray) [inline oracle/oracle-instantclient-12.1-x86_64.spec]: corrected your recommandation in the last commit of the same branch

**Comment by remicollet** [comment]
Tanks a lot.

Merged

**Pull request closed without merging**